### ISSUE TITLE
Crypto-js not working with writeFile?

### ISSUE THREAD

**Issue by Nabeel20** [alpha0010/react-native-file-access#11]
I am trying to use AES encryption with a text, decryption the text is ok but I got an error when trying to write text to file. 
```
let encryptedAES = CryptoJS.AES.encrypt("Message", "My Secret Passphrase").toString();
```
I pass encryptedAES to writeFile but it fails.
Any help?

**Comment by alpha0010** [comment]
I just published 1.1.0 which supports base 64 encoded strings.  React Native sometimes has errors when passing binary strings, so that may fix your issue.

If that still does not work, could you post the error message and a code snippet with the APIs you are using from this library?

**Comment by Nabeel20** [comment]
Thank you for base64, and I think it its a problem with ```crypto-js "4.0.0" ``` so I downgrade to version ``3.3.0`` and now AES encryption works fine!
